src/main/java/org/prebid/server/auction/BidResponseCreator.java

```diff
@@ -222,7 +222,8 @@ private static boolean isEmptyBidderResponses(List<BidderResponse> bidderRespons
                 auctionContext,
                 cacheInfo,
                 eventsContext)
-                .compose(cacheResult -> videoStoredDataResult(bidRequest.getImp(), auctionContext.getTimeout())
+                .compose(cacheResult -> videoStoredDataResult(auctionContext.getAccount(), bidRequest.getImp(),
```

Comment:
Minor: `videoStoredDataResult` could accept `AuctionContext` and pull account, imps and timeout from there. This will make call site look neat.